### PR TITLE
NEPT-149: update contributing.md

### PR DESCRIPTION
**Pull request by voidtek**
https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-149

### CODE REVIEW

**Comment by dxvargas** [comment]
👎 
Also contributing.json must be updated.
Since we are not following gitmagic standards, we should just delete contributing files (md and json).

**Comment by dxvargas** [comment]
👍

**Comment by gillesdeudon** [comment]
:+1:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,50 +2,28 @@
...
-<type>(<scope>): <subject>
-<BLANK LINE>
-<body>
+NEPT-[ticket_number]: [commit_description]
 ```
 
-The **header** is mandatory and the **scope** of the header is optional.
-
-Any line of the commit message cannot be longer 100 characters! This allows the message to be easier
+The commit description cannot be longer 100 characters! This allows the message to be easier
```
Comment (jfhovinne) [inline CONTRIBUTING.md]: "... cannot be longer than ..."

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,50 +2,28 @@
...
+If the commit reverts a previous commit, it should begin with `revert: `, followed by the reverted commit.
 
-### Scope
-The scope could be any component the commit is affecting. Use the business components
-defined at https://webgate.ec.europa.eu/CITnet/confluence/display/NEXTEUROPA/WCM+Components.
-
-### Subject
-The subject contains succinct description of the change:
+### Commit Description
+The description contains succinct description of the change:
```
Comment (jfhovinne) [inline CONTRIBUTING.md]: "The description contains a succinct description of the change and should follow below rules:"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,50 +2,28 @@
...
 
 * use the imperative, present tense: "change" not "changed" nor "changes"
 * don't capitalize first letter
```
Comment (jfhovinne) [inline CONTRIBUTING.md]: IMO first letter should be in uppercase.

**Comment by dxvargas** [comment]
👍

**Comment by gillesdeudon** [comment]
:+1:

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -2,50 +2,29 @@
...
+the following rules:
+
+* Use the imperative, present tense: "change" not "changed" nor "changes"
+* Capitalise first letter
+* No dot (.) at the end of the sentence
+
+### Branch Name Format
+
+```
+nept-[ticket_name]-[base_branch]-[additional_information]
```
Comment (jfhovinne) [inline CONTRIBUTING.md]: [ticket_name] should be replaced by [ticket_number], so we use the same wording as above.

**Comment by gillesdeudon** [comment]
:+1:

**Comment by dxvargas** [comment]
👍

**Pull request merged**